hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2083,7 +2083,9 @@
       name: chip_sw_entropy_src_cs_aes_halt
       desc: '''Verify the aes halt handshake with CSRNG.
 
-            Details TBD.
+            - Add a top level assertion to check that aes halts and waits for
+              entropy when it's unavaliable on the edn.
+            - Modify the edn_req_test as necessary to hit this case and cover the above assertion.
```

Comment:
Hey @drewmacrae , the plan looks good to me but this test plan entry now looks more like a TODO list than a description of the envisioned test.

I am okay to merge this as is and then the description can be reworked based on how the test looks exactly eventually.